test/e2e/specs/onboarding/vertical__none.ts

```diff
@@ -0,0 +1,140 @@
+				testUser.email,
+				testUser.username,
+				testUser.password
+			);
+
+			userCreatedFlag = true;
+		} );
+
+		it( 'Select a free .wordpress.com domain', async function () {
+			domainSearchComponent = new DomainSearchComponent( page );
+			await domainSearchComponent.search( testUser.siteName );
+			await domainSearchComponent.selectDomain( '.wordpress.com' );
+		} );
+
+		it( 'Select WordPress.com Free plan', async function () {
+			const signupPickPlanPage = new SignupPickPlanPage( page );
+			await signupPickPlanPage.selectPlan( 'Free' );
+		} );
+
+		it( 'Skip goals step', async function () {
+			startSiteFlow = new StartSiteFlow( page );
+
+			await page.waitForLoadState( 'networkidle' );
+			const currentStep = await startSiteFlow.getCurrentStep();
+
+			if ( currentStep === 'goals' ) {
+				await startSiteFlow.clickButton( 'Continue' );
+			}
+		} );
+	} );
+
+	describe( 'Vertical flow', function () {
+		it( 'Skip vertical selection', async function () {
+			const currentStep = await startSiteFlow.getCurrentStep();
+			if ( currentStep === 'vertical' ) {
+				await startSiteFlow.clickButton( 'Continue' );
+			}
+		} );
+
+		// Validate that theme picker is used instead of vertical design screen
+		it( 'See theme picker screen for no vertical', async function () {
+			await startSiteFlow.validateOnDesignPickerScreen();
+		} );
+
+		it( 'Navigate back for Something else flow', async function () {
+			await startSiteFlow.goBackOneScreen();
+		} );
+
+		it( 'Select Something else', async function () {
+			const currentStep = await startSiteFlow.getCurrentStep();
+			if ( currentStep === 'vertical' ) {
+				await startSiteFlow.enterVertical( 'People & Society' );
+				await startSiteFlow.selectVertical( 'Something else' );
+				await startSiteFlow.clickButton( 'Continue' );
+			}
+		} );
+
+		// Validate that theme picker is used instead of vertical design screen
+		it( 'See theme picker screen for Something else vertical', async function () {
+			await startSiteFlow.validateOnDesignPickerScreen();
+		} );
+
+		it( 'Navigate back for skip to dashboard flow', async function () {
+			await startSiteFlow.goBackOneScreen();
+		} );
+
+		// TODO: Go through theme selection flow
+		it( 'Skip to dashboard', async function () {
+			await startSiteFlow.clickButton( 'Skip to dashboard' );
+		} );
+
+		// TODO: Investigate bug where last selected vertical is used even when
+		// vertical is eventually skipped
+		it( 'See site on home dashboard', async function () {
+			const myHomePage = new MyHomePage( page );
+			await myHomePage.validateSiteTitle( 'Something else' );
+		} );
+	} );
+
+	afterAll( async function () {
+		const restAPIClient = new RestAPIClient(
+			{ username: testUser.username, password: testUser.password },
+			userDetails.body.bearer_token
+		);
+
+		if ( userCreatedFlag ) {
+			await apiCloseAccount( restAPIClient, {
+				userID: userDetails.body.user_id,
+				username: testUser.username,
+				email: testUser.email,
```

Comment:
**praise**: Good work including the cleanup here.

Reply:
Thank you!! Unfortunately, this part was eliminated when started using existing users, but I appreciate the praise.